app/models/variable.rb

```diff
@@ -76,6 +76,18 @@ def criteria?
     values.any? { |value| value.is_criteria? }
   end
 
+  def self.feedback_like
+    find_by(name: 'feedback_like')
```

Comment:
@radinreth what if the variable_name is not 'feedback_like'

Reply:
we can use :mark_as

because this variable is created just to setup simulate data that's why to keep it simple i query by name instead.

Reply:
this is in clone_step, and in reality, we don't have clone step for these variables yet